sympy/geometry/line.py

```diff
@@ -36,16 +46,14 @@ class Undecidable(ValueError):
 
 
 class LinearEntity(GeometrySet):
-    """A base class for all linear entities (line, ray and segment)
-    in a 2-dimensional Euclidean space.
+    """A base class for all linear entities (Line, Ray and Segment)
+    in n-dimensional Euclidean space.
 
     Attributes
     ==========
 
     p1
     p2
-    coefficients
```

Comment:
Why are these deleted? Beside `p1, p2, and points` there are also `ambient_dimension, coefficients, direction and length`.